roles/cinder-data/tasks/main.yml

```diff
@@ -22,12 +22,15 @@
             dest=/etc/cinder/cinder.encryption.conf mode=0640
             owner=root group=cinder
   notify: restart cinder services
-  when: cinder.fixed_key is defined
+  when:
```

Comment:
Not sure if we still support fixed_key.
If not, we can remove this task.
If yes, we may need to keep fixed_key configuration and barbican configuration at the same place. Either in cinder.conf or cinder.encryption.conf
i.e. in the template
```
{% if barbican.enabled %}
...
{% elif fixed_key.enabled %}
...
{% endif %}